transforms/.make.transforms

```diff
@@ -224,7 +224,7 @@ conventions:
 
 # Expects RUN_FILE
 .PHONY: .transforms.run-src-file
-.transforms.run-src-file:
+.transforms.run-src-file: .defaults.run-src-file
```

Comment:
should the body of this target be removed now that it calls .defaults?

Reply:
otherwise, i think this will run twice.

Reply:
yes , removed the body and tested. works fine.

Reply:
please mark associated issues fixed.